hbp_validation_framework/datastores.py

```diff
@@ -263,10 +263,25 @@ class SwiftDataStore(object):
     def __init__(self, **kwargs):
         pass
 
-    def upload_data(self, file_paths):
-        raise NotImplementedError("The SwiftDataStore does not support uploading data.")
-
-    def get_container(self, remote_path):
+    def upload_data(self, file_paths, username="", container="", project=None, remote_directory="", overwrite=False):
+        try:
+            from hbp_archive import Container
+        except ImportError:
+            print("Please install the following package: hbp_archive")
+            return
+        
+        print("----------------------------------------------------")
+        print("NOTE: The target location is inside a CSCS container")
+        print("----------------------------------------------------")
+        if not username:
+            username = raw_input("Please enter your CSCS username: ")
```

Comment:
This is Python 2 specific. In Python 3, `input` has the same behaviour as `raw_input` in Python 2

Reply:
Done now.